modules/ding_permissions/ding_permissions.module

```diff
@@ -503,9 +503,11 @@ function ding_permissions_secure_permissions($role) {
       'administer place2book settings',
       'administer shortcuts',
       'administer url aliases',
+      'administer users',
```

Comment:
Adding the administer users permission to editors seems like a bad idea - even though that is what is suggested by the business. It provides to wide permissions compared to what they need. I have asked them to reconsider this.